validation-test/stdlib/RangeReplaceable.swift.gyb

```diff
@@ -112,6 +112,36 @@ RangeReplaceableTestSuite.test("replaceSubrange/countableClosedRange") {
   }
 }
 
+RangeReplaceableTestSuite.test("removeSubrange/countableRange") {
+  for test in removeRangeTests {
+    var c = 
+      MinimalRangeReplaceableRandomAccessCollectionWithStrideableIndex(elements: test.collection)
+    let rangeToRemove = test.rangeSelection.countableRange(in: c)
+    c.removeSubrange(rangeToRemove)
+    expectEqualSequence(
+      test.expected,
+      c.map { $0.value },
+      stackTrace: SourceLocStack().with(test.loc))
+  }
+}
+
+RangeReplaceableTestSuite.test("removeSubrange/countableClosedRange") {
+  for test in removeRangeTests {
+    switch test.rangeSelection {
+      case .emptyRange: continue
+      default: break
+    }
+    var c = 
+      MinimalRangeReplaceableRandomAccessCollectionWithStrideableIndex(elements: test.collection)
+    let rangeToRemove = test.rangeSelection.countableClosedRange(in: c)
+    c.removeSubrange(rangeToRemove)
+    expectEqualSequence(
+      test.expected,
+      c.map { $0.value },
+      stackTrace: SourceLocStack().with(test.loc))
+  }
+}
+
```

Comment:
I also think these 2 tests can be GYBbed together. They are only different in test name and test field they use to get the range to remove.